public/workers/pyright-a8ec35e78b1967df3217.worker.js.LICENSE.txt

```diff
@@ -0,0 +1,8 @@
+/*!
+ * The buffer module from node.js, for the browser.
+ *
+ * @author   Feross Aboukhadijeh <https://feross.org>
+ * @license  MIT
+ */
+
+/*! ieee754. BSD-3-Clause License. Feross Aboukhadijeh <https://feross.org/opensource> */
```

Comment:
Hah, this is clearly inadequate. It's just what the webpack build splits out. We'd be better with some explicit credit for Pyright.

I'd also like to look at publishing a package for it rather than repeatedly splatting 1.3MB of JS in this repo. That'd be a good chance to improve this.

Reply:
Sorted on #251